apparmor.d/groups/kde/kwin_wayland

```diff
@@ -78,10 +95,20 @@ profile kwin_wayland @{exec_path} flags=(attach_disconnected mediate_deleted) {
   @{run}/udev/data/c13:@{int}  r,         # for /dev/input/*
   @{run}/udev/data/c226:@{int} r,         # for /dev/dri/card*
 
+  @{run}/udev/data/+usb:* r,
+  @{run}/udev/data/c189:@{int} r,
+  @{run}/udev/data/+pci@{int}/**/ r,
+  @{run}/udev/data/+pci@{int}/**/f.3 r,
```

Comment:
This paths do not exist (there is no subfolder in `udev/data`, and they will never be). You might have confused with `:` or `.`
The correct rule is:
```
@{run}/udev/data/+pci:* r,
```

Reply:
I see. This was proposed by aa-log